### DISCUSSION TITLE
Bending and Tensile Stress Test

### DISCUSSION THREAD

**Discussion by sreejitsarmac** [idaholab/moose#25851] [Q&A Getting Started]
How would you, from scratch, create a beam or a bar, mesh it, and find the bending deformation, bending stress, tensile deformation, and tensile stress? I'm an absolute newbie to MOOSE so please give a very detailed explanation with code. Thanks!

**Comment by GiudGiud** [comment]
There should be an example somewhere. Can you look in the tensor mechanics module?
